vignettes/Getting_Started.Rmd

```diff
@@ -543,13 +544,14 @@ Explicit calculation yields the same:
 ```{r}
 a <- SteppedPower::wlsPower(Cl=rep(6,4), mu0=mu0, mu1=mu1, AR=AR,
                        sigma=0, tau=tau, N=N, psi=psi, verbose=2, INDIV_LVL = TRUE)
+a
```

Comment:
This is what I mean with the extra "printout line". 
I think it helps the user to grasp what she/he can really get out with SteppedPower

Reply:
Some other potential sweet spots are marked in the email